ui/pages/confirmations/components/confirm/info/base-transaction-info/base-transaction-info.tsx

```diff
@@ -17,6 +20,9 @@ const BaseTransactionInfo = () => {
 
   return (
     <>
+      {transactionMeta.origin === DAPP_SWAP_COMPARISON_ORIGIN && (
```

Comment:
Is it cleaner to encapsulate this origin knowledge inside the `DappSwapComparisonBanner` itself so it can handle its own conditional rendering?

Reply:
The reason I put this condition here is that even if I keep it inside `DappSwapComparisonBanner` we will end up running few hooks like get decimal places for erc20, get USD rates, etc for a lot of confirmationations.

By keeping this condition here all that code execution is by-passed.